stable/prometheus-rabbitmq-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 0.4.0
+version: 0.5.0
```

Comment:
Should be `0.4.1` since there's no new features.

Reply:
Done!